Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Stackable",
+    platforms: [
+        .iOS(.v12),
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Stackable",
```

Comment:
Worth noting that the cocoapod is named `RPStackable`. Not sure what the right answer is for SPM.

Reply:
Is the idea to prevent name collisions? I've changed the package name to `RPStackable` for consistency, but the generated library will still be called `Stackable`.

Reply:
Yeah `Stackable` was taken on cocoapods unfortunately.
Sweet -  that is in sync with cocoapods.